src/libsystemd-network/test-ndisc-rs.c

```diff
@@ -193,6 +193,11 @@ int icmp6_bind_router_solicitation(int index) {
         return test_fd[0];
 }
 
+int icmp6_bind_router_advertisment(int index) {
```

Comment:
Shouldn't `index` be unsigned?

Looking at subsequent patches, maybe it'd be nicer to return the `fd` in an output argument: `icmp6_bind_router_advertisment(unsigned index, int *fd)`? This would make life easier for the caller because it a pointer to the field for the fd could passed, and the caller wouldn't need to move the value around. Just an idea.